src/MimeType.js

```diff
@@ -0,0 +1,38 @@
+export const supportedTypes = ['ifc', 'obj']
+
+
+/**
+ * @param {string} ext
+ * @return {boolean} Is supported
+ */
+export function isExtensionSupported(ext) {
+  return ext.match(/(?:ifc|obj)/i)
+}
+
+
+/**
+ * @param {string} strWithSuffix
+ * @return {boolean} Is supported
+ */
+export function pathSuffixSupported(pathWithSuffix) {
+  const lastDotNdx = pathWithSuffix.lastIndexOf('.')
+  if (lastDotNdx === -1) {
+    return false
+  }
+  return isExtensionSupported(pathWithSuffix.substring(lastDotNdx + 1))
+}
+
+
+/**
+ * @param {string} filepath
+ * @return {Array.<string>}
```

Comment:
This function returns an object that consists of two members which are arrays.

Reply:
Done